spec/latest/json-ld/index.html

```diff
@@ -1413,26 +1413,54 @@ <h2>Embedding</h2>
     <a>property</a> values. This is a commonly used mechanism for
     creating a parent-child relationship between two <a>nodes</a>.</p>
 
-  <p>The example shows two nodes related by a property from the first node:</p>
+  <p>Without embedding, <a>node objects</a> can be linked by referencing the
+    identifier of another <a>node object</a>. For example:</p>
+
+  <pre class="example nohighlight" data-transform="updateExample"
+       title="Referencing node objects">
+  <!--
+  [{
+    "@context": {
+      "@vocab": "http://schema.org/",
+      ****"knows": {"@type": "@id"}****
+    },
+    "name": "Manu Sporny",
+    "@type": "Person",
+    ****"knows": "http://foaf.me/gkellogg#me"****
+  }, {
+    "@id": "http://foaf.me/gkellogg#me",
+    "@type": "Person",
+    "name": "Gregg Kellogg"
```

Comment:
I took another look at the latest rendered spec with this update and noticed that the `@context` from other object doesn't seem to apply here and `name` doesn't have mapping to IRI. Does this object need something like `"@context": { "@vocab": "http://schema.org/" }`?